cryptus-app/components/profile/viewer_profile/viewer_profile.module.scss

```diff
@@ -0,0 +1,28 @@
+@use "/styles/variables" as v;
+
+.container {
+  flex: 1 1 auto;
+
+  flex-direction: column;
+  align-items: center;
+}
+
+.wallet { //Placeholder
+  height: 800px; //Just to see it scroll
+  margin-bottom: v.$footer-h;
+  width: 100%;
```

Comment:
width is not 100% when rendered:
![image](https://user-images.githubusercontent.com/32850285/152713570-db693620-e489-4a5e-b38e-79f507873a7c.png)

Reply:
true j'ai changé le nom du container dans creator et pas dans viewer